core/schema/container.go

```diff
@@ -81,11 +81,19 @@ func (s *containerSchema) Install() {
 		dagql.Func("directory", s.directory).
 			Doc(`Retrieves a directory at the given path.`,
 				`Mounts are included.`).
-			ArgDoc("path", `The path of the directory to retrieve (e.g., "./src").`),
+			ArgDoc("path", `The path of the directory to retrieve (e.g., "./src").`).
+			ArgDoc("expand",
+				"Replace `${VAR}` or `$VAR` in the value of path according to the current "+
+					`environment variables defined in the container (e.g.,
+				"/$VAR/foo").`),
```

Comment:
nit: formatting.

You can just use backtick strings for all of this.

Reply:
the doc comment itself has backticks, so maybe we can keep it this way? (that way it also stays consistent with what we have in ExpandEnvVariable)

Reply:
Hi @jedevc, I have merged the 2nd and 3rd line into a single line. The first line includes backticks in the string itself, so I don't think we can include complete line into single backtick'ed string. 

Kindly let me know if you have any alternate recommendation for handling this.